playbooks/roles/streisand-gateway/templates/instructions.md.j2

```diff
@@ -4,9 +4,16 @@
 {% endfor %}
 - - -
 
-Your Streisand Gateway contains step-by-step instructions for the services it provides, and mirrors of all necessary client software. This document covers the SSL Certificate installation instructions and will show you how to connect to the Gateway to learn how to configure clients to use the Streisand services.
+Your Streisand Gateway contains step-by-step instructions for the services it provides, and mirrors of all necessary client software.
+
+{% if le_ok %}
+This document will show you how to connect to the Gateway to learn how to configure clients to use the Streisand services.
+{% else %}
+This document covers the SSL Certificate installation instructions and will show you how to connect to the Gateway to learn how to configure clients to use the Streisand services.
+{% endif %}
 
 ---
+{% if not le_ok %}
```

Comment:
@cpu Hmm. As you could see, instructions about self-signed certificates are only rendered if `le_ok` is not true.

Reply:
You're right! I think I was looking at the wrong generated-docs file. Apologies. A second run with d4ca89a produced a generated docs file with no CA import instructions and the correct HTTPS URL :+1: